README.md

```diff
@@ -362,6 +362,31 @@ It tries to keep it simple and has the following priorities in mind:
 2. Easy for the **developer** who process them using the exposed APIs.
 3. Easy for the **maintainer/contributor** who maintain the `easy-template-x` package itself.
 
+## Extensions
```

Comment:
Once you'll publish your extensions to npm/github I think it will be great link to it from the readme.

Reply:
That was the plan.